images/linux/scripts/SoftwareReport/SoftwareReport.Generator.ps1

```diff
@@ -0,0 +1,221 @@
+    (Get-HGVersion),
+    (Get-MinikubeVersion),
+    (Get-NewmanVersion),
+    (Get-NvmVersion),
+    (Get-PackerVersion),
+    (Get-PhantomJSVersion),
+    (Get-SwigVersion),
+    (Get-TerraformVersion),
+    (Get-UnZipVersion),
+    (Get-WgetVersion),
+    (Get-ZipVersion),
+    (Get-ZstdVersion)
+)
+
+if (-not (Test-IsUbuntu16)) {
+    $toolsList += @(
+        (Get-PodManVersion),
+        (Get-BuildahVersion),
+        (Get-SkopeoVersion)
+    )
+}
+
+$markdown += New-MDList -Style Unordered -Lines ($toolsList | Sort-Object)
+
+$markdown += New-MDHeader "CLI Tools" -Level 3
+$markdown += New-MDList -Style Unordered -Lines @(
+    (Get-AlibabaCloudCliVersion),
+    (Get-AWSCliVersion),
+    (Get-AWSCliSessionManagerPluginVersion),
+    (Get-AWSSAMVersion),
+    (Get-AzureCliVersion),
+    (Get-AzureDevopsVersion),
+    (Get-GitHubCliVersion),
+    (Get-HubCliVersion),
+    (Get-NetlifyCliVersion),
+    (Get-OCCliVersion),
+    (Get-ORASCliVersion),
+    (Get-VerselCliversion)
+)
+
+$markdown += New-MDHeader "Java" -Level 3
+$markdown += Get-JavaVersions | New-MDTable
+$markdown += New-MDNewLine
+
+$markdown += New-MDHeader "PHP" -Level 3
+$markdown += Build-PHPTable | New-MDTable
+$markdown += New-MDNewLine
+
+$markdown += New-MDHeader "Haskell" -Level 3
+$markdown += New-MDList -Style Unordered -Lines @(
+    (Get-GHCVersion),
+    (Get-CabalVersion),
+    (Get-StackVersion)
+)
+
+$markdown += New-MDHeader "Rust Tools" -Level 3
+$markdown += New-MDList -Style Unordered -Lines @(
+    (Get-RustVersion),
+    (Get-RustupVersion),
+    (Get-RustdocVersion),
+    (Get-CargoVersion)
+)
+
+$markdown += New-MDHeader "Packages" -Level 4
+$markdown += New-MDList -Style Unordered -Lines @(
+    (Get-BindgenVersion),
+    (Get-CargoAuditVersion),
+    (Get-CargoOutdatedVersion),
+    (Get-CargoClippyVersion),
+    (Get-CbindgenVersion),
+    (Get-RustfmtVersion)
+)
+
+$markdown += New-MDHeader "Browsers and Drivers" -Level 3
+$markdown += New-MDList -Style Unordered -Lines @(
+    (Get-ChromeVersion),
+    (Get-ChromeDriverVersion),
+    (Get-FirefoxVersion),
+    (Get-GeckodriverVersion)
+)
+
+$markdown += New-MDHeader ".NET Core SDK" -Level 3
+$markdown += New-MDList -Style Unordered -Lines @(
+    (Get-DotNetCoreSdkVersions)
+)
+
+$markdown += New-MDHeader "Az Module" -Level 3
+$markdown += New-MDList -Style Unordered -Lines @(
+    (Get-AzModuleVersions)
+)
+
+$markdown += New-MDHeader "Databases" -Level 3
+$markdown += New-MDList -Style Unordered -Lines @(
+    (Get-PostgreSqlVersion),
+    (Get-MongoDbVersion),
+    (Get-SqliteVersion)
+)
+
+$markdown += New-MDHeader "MySQL" -Level 4
+$markdown += New-MDList -Style Unordered -Lines @(
+    (Get-MySqlVersion),
+    "MySQL Server (user:root password:root)",
+    "MS SQL Server Client Tools"
+)
+$markdown += New-MDCode -Lines @(
+    "MySQL service is disabled by default. Use the following command as a part of your job to start the service: 'sudo systemctl start mysql.service'"
```

Comment:
Maybe move it to Databases file out of common generator that just builds the document?